platform/android/MapboxGLAndroidSDK/src/main/java/org/maplibre/android/style/expressions/Expression.java

```diff
@@ -1565,18 +1565,46 @@ public static Expression indexOf(@NonNull Expression keyword, @NonNull Expressio
   }
 
   /**
-   * Returns the first position at which a `needle` can be found in a `haystack`. 
+   * Returns items from an array or a substring from a string from a specified start index. 
+   * The return value is inclusive of the start index.
    *
    * @param needle   the item expression
    * @param haystack the array or string expression
    * @param fromIndex the index to start searching from
-   * @return position in the array or string or -1 if not found.
+   * @return array or string
```

Comment:
Is this correct?

Reply:
Ah it is not… Somehow I put it here by mistake. Feel free to revert this line on your own or I will do that nearest days. Thanks for noticing it!

Reply:
Sorry, I simply forgot about this 😄 I fixed it. Hope CI will pass 🤞🏻